### PR TITLE
Release v0.12.0

### PR DESCRIPTION
**Pull request by ChristophWurst**
I think we are feature complete with v0.11 for the most part. Anything missing or broken will be fixed in follow-up releases. As someone that usually tells people to follow *release early, release often*, I can no longer wait with this :wink: 

[Release checklist](https://blog.wuc.me/2018/08/14/nextcloud-app-release-process.html)
- [x] Created release branch.
- [x] Updated version number.
  - [x] Updated `appinfo/info.xml`.
  - [x] Updated `package.json` and ran `npm install`.
- [x] [Alpha 1](https://github.com/nextcloud/mail/releases/tag/v0.12.0-alpha1)
- [x] [Alpha 2](https://github.com/nextcloud/mail/releases/tag/v0.12.0-alpha2)
- [x] [Alpha 3](https://github.com/nextcloud/mail/releases/tag/v0.12.0-alpha3)
- [x] [Alpha 4](https://github.com/nextcloud/mail/releases/tag/v0.12.0-alpha4)
- [x] [Alpha 5](https://github.com/nextcloud/mail/releases/tag/v0.12.0-alpha5)
- [x] [Beta 1](https://github.com/nextcloud/mail/releases/tag/v0.12.0-beta1)
- [x] [Beta 2](https://github.com/nextcloud/mail/releases/tag/v0.12.0-beta2)
- [x] [Beta 3](https://github.com/nextcloud/mail/releases/tag/v0.12.0-beta3)
- [x] [Beta 4](https://github.com/nextcloud/mail/releases/tag/v0.12.0-beta4)
- [x] [Beta 5](https://github.com/nextcloud/mail/releases/tag/v0.12.0-beta5)
- [x] [RC 1](https://github.com/nextcloud/mail/releases/tag/v0.12.0-RC1)
- [x] [RC 2](https://github.com/nextcloud/mail/releases/tag/v0.12.0-RC2)
- [x] [RC 3](https://github.com/nextcloud/mail/releases/tag/v0.12.0-RC3)
- [x] Updated the changelog.
- [ ] Created release tag.
- [ ] Pushed tag to GitHub.
- [ ] Built package with `krankerl package`.
- [ ] Created new GitHub release.
- [ ] Published app on App Store.

My current plan is to release the week after next.

cc @nextcloud/mail

### CODE REVIEW

**Comment by ChristophWurst** [comment]
so, @nextcloud/mail, any feedback? bugs? regressions?

**Comment by ChristophWurst** [comment]
@nextcloud/mail could you please test the mobile version? Do you see a burger icon wo show the navigation?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 0.12.0 - 2018-04-08
+### Added
+- Nextcloud 16 support
+- A new front-end (with lots of improvements)
+- Many UX enhancements
+- New and updated translations
```
Comment (J0WI) [inline CHANGELOG.md]: PHP 7.3 support

Comment (ChristophWurst) [reply CHANGELOG.md]: yes, thank you. added :)

**Pull request merged**